### PR TITLE
TiKV Config: Update some optional values of TiKV configuration items

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Updated the value of `wal-recovery-mode`, `rate-limiter-mode`, `compaction-pri`, and `compaction-style`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7376
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1187,11 +1182,16 @@ bloom filter 为每个 key 预留的长度。
...
-+ `defaultcf` 默认值：`3`
-+ `writecf` 默认值：`3`
-+ `lockcf` 默认值：`1`
++ 优先处理 compaction 的类型
++ 可选值：
+    + `"by-compensated-size"`：根据大小顺序，优先对大的文件进行 compaction
+    + `"oldest-largest-seq-first"`：根据时间顺序，优先对数据的最新更新时间晚的文件进行 compaction。当你只在小范围内更新部分热键 (hot keys) 时，可以使用此配置。
+    + `"oldest-smallest-seq-first"`：根据时间顺序，优先对长时间没有被 compaction 到下一级的文件进行 compaction。如果你在 Key 空间内随机更新了 hot key，此时设置该值可以轻微缓解写放大。
+    + `"min-overlapping-ratio"`：根据重叠比例，优先对在不同层内文档大小重叠比例高的文件进行 compaction，即 `文档在下一层时的大小`/`该文档在本层时的大小` 的值越小，compaction 的优先级越高。在许多场景下，该配置可以有效缓解写放大。
+    + 的文件进行 compaction。
```
Comment (tabokie) [inline tikv-configuration-file.md:1191]:
```suggestion
    + `"by-compensated-size"`：根据大小顺序，优先对大的文件进行 compaction。
    + `"oldest-largest-seq-first"`：根据时间顺序，优先对数据的最新更新时间晚的文件进行 compaction。当你只在小范围内更新部分热键 (hot keys) 时，可以使用此配置。
    + `"oldest-smallest-seq-first"`：根据时间顺序，优先对长时间没有被 compaction 到下一级的文件进行 compaction。如果你在 Key 空间内随机更新了 hot key，此时设置该值可以轻微缓解写放大。
    + `"min-overlapping-ratio"`：根据重叠比例，优先对在不同层内文档大小重叠比例高的文件进行 compaction，即 `文档在下一层时的大小`/`该文档在本层时的大小` 的值越小，compaction 的优先级越高。在许多场景下，该配置可以有效缓解写放大。
```

**Review by tabokie — APPROVED** [review]

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1187,11 +1182,15 @@ bloom filter 为每个 key 预留的长度。
...
-+ 可选择值：`0` (`ByCompensatedSize`)，`1` (`OldestLargestSeqFirst`)，`2` (`OldestSmallestSeqFirst`)，`3` (`MinOverlappingRatio`)。
-+ `defaultcf` 默认值：`3`
-+ `writecf` 默认值：`3`
-+ `lockcf` 默认值：`1`
++ 优先处理 compaction 的类型
++ 可选值：
+    + `"by-compensated-size"`：根据大小顺序，优先对大的文件进行 compaction。
+    + `"oldest-largest-seq-first"`：根据时间顺序，优先对数据的最新更新时间晚的文件进行 compaction。当你只在小范围内更新部分热键 (hot keys) 时，可以使用此配置。
+    + `"oldest-smallest-seq-first"`：根据时间顺序，优先对长时间没有被 compaction 到下一级的文件进行 compaction。如果你在 Key 空间内随机更新了 hot key，此时设置该值可以轻微缓解写放大。
+    + `"min-overlapping-ratio"`：根据重叠比例，优先对在不同层内文档大小重叠比例高的文件进行 compaction，即 `文档在下一层时的大小`/`该文档在本层时的大小` 的值越小，compaction 的优先级越高。在许多场景下，该配置可以有效缓解写放大。
```
Comment (tabokie) [inline tikv-configuration-file.md:1190]:
```suggestion
    + `"by-compensated-size"`：根据大小顺序，优先对大文件进行 compaction。
    + `"oldest-largest-seq-first"`：根据时间顺序，优先对数据更新时间晚的文件进行 compaction。当你只在小范围内更新部分热点键 (hot keys) 时，可以使用此配置。
    + `"oldest-smallest-seq-first"`：根据时间顺序，优先对长时间没有被 compact 到下一级的文件进行 compaction。如果你在大范围内随机更新了部分热点键，使用该配置可以轻微缓解写放大。
    + `"min-overlapping-ratio"`：根据重叠比例，优先对在不同层间文件重叠比例高的文件进行 compaction，即 `下一层相关文件大小`/`本层相关文件大小` 的值越小，compaction 的优先级越高。在许多场景下，该配置可以有效缓解写放大。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -887,16 +887,13 @@ rocksdb 相关的配置项。
...
-+ 3 (SkipAnyCorruptedRecords)：灾难后恢复。跳过日志中损坏的记录，尽可能多的恢复数据。
-
-+ 默认值：2
-+ 最小值：0
-+ 最大值：3
++ 预写式日志 (WAL, Write Ahead Log) 的恢复模式。
++ 可选值：
+    + `"tolerate-corrupted-tail-records"`：容忍并丢弃位于日志尾部的不完整的数据 (trailing data)
+    + `"absolute-consistency"`：当发现待恢复的日志中有被损坏的日志时，放弃恢复所有日志
+    + `"point-in-time"`：按顺序恢复日志，直到遇到第一个损坏的日志后再停止恢复剩余的日志
```
Comment (TomShawn) [inline tikv-configuration-file.md:894]:
```suggestion
    + `"point-in-time"`：按顺序恢复日志。遇到第一个损坏的日志时，停止恢复剩余的日志
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -887,16 +887,13 @@ rocksdb 相关的配置项。
...
-+ 2 (PointInTimeRecovery)：按顺序恢复日志，直到碰到第一个损坏的记录。
-+ 3 (SkipAnyCorruptedRecords)：灾难后恢复。跳过日志中损坏的记录，尽可能多的恢复数据。
-
-+ 默认值：2
-+ 最小值：0
-+ 最大值：3
++ 预写式日志 (WAL, Write Ahead Log) 的恢复模式。
++ 可选值：
+    + `"tolerate-corrupted-tail-records"`：容忍并丢弃位于日志尾部的不完整的数据 (trailing data)
+    + `"absolute-consistency"`：当发现待恢复的日志中有被损坏的日志时，放弃恢复所有日志
```
Comment (TomShawn) [inline tikv-configuration-file.md:893]:
```suggestion
    + `"tolerate-corrupted-tail-records"`：容忍并丢弃位于日志尾部的不完整的数据 (trailing data)。
    + `"absolute-consistency"`：当发现待恢复的日志中有被损坏的日志时，放弃恢复所有日志。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1187,11 +1182,15 @@ bloom filter 为每个 key 预留的长度。
...
-+ `defaultcf` 默认值：`3`
-+ `writecf` 默认值：`3`
-+ `lockcf` 默认值：`1`
++ 优先处理 compaction 的类型
++ 可选值：
+    + `"by-compensated-size"`：根据大小顺序，优先对大文件进行 compaction。
+    + `"oldest-largest-seq-first"`：根据时间顺序，优先对数据更新时间晚的文件进行 compaction。当你只在小范围内更新部分热点键 (hot keys) 时，可以使用此配置。
+    + `"oldest-smallest-seq-first"`：根据时间顺序，优先对长时间没有被 compact 到下一级的文件进行 compaction。如果你在大范围内随机更新了部分热点键，使用该配置可以轻微缓解写放大。
+    + `"min-overlapping-ratio"`：根据重叠比例，优先对在不同层之间文件重叠比例高的文件进行 compaction，即一个文件在 `下一层的大小`/`本层的大小` 的值越小，compaction 的优先级越高。在诸多场景下，该配置可以有效缓解写放大。
++ 默认值：
```
Comment (TomShawn) [inline tikv-configuration-file.md:1191]: 这部分的内容输入来自？

Comment (en-jin19) [reply tikv-configuration-file.md:1191]: 请查看 https://github.com/pingcap/docs/pull/7376/files 中新增的 comments，感谢~！

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**